script/incremental_build.sh

```diff
@@ -6,6 +6,12 @@ readonly REPO_DIR="$(dirname "$SCRIPT_DIR")"
 
 source "$SCRIPT_DIR/common.sh"
 
+if [ "$(expr substr $(uname -s) 1 5)" == "MINGW" ]; then
+  PUB=pub.bat
+else
+  PUB=pub
+fi
```

Comment:
Could the if here just be `command -v pub.bat`?

Reply:
What's the advantage of that approach? If for some reason there were an executable called `pub.bat` in the path on Linux or macOS we wouldn't want to run it, and if `pub.bat` isn't there on Windows `pub` still wouldn't be the right command to try to run, so it seems like an odd construction to me.

Reply:
I'm worried if the Windows image changes this will break. But that should be covered by CI.  LGTM.

Reply:
Hopefully we'll rewrite it in Dart before then :)

Reply:
You are much more optimistic than I am :D